contextualized/regression.py

```diff
@@ -416,6 +419,9 @@ def test_step(self, batch, batch_idx):
         self.log_dict({'test_loss': loss})
         return loss
 
+    def predict_from_sample_models(self, X, beta_hat, mu_hat):
```

Comment:
`predict_from_models`

Reply:
Why private? I can imagine post-hoc analyses that may want to use this function (e.g. sensitivity anlalyses) and it's not clear to me why it should be private.